src/valves.js

```diff
@@ -13,15 +13,18 @@ import { MiscUtils } from './miscUtils';
 
 var Valves = EventEmitter(function () {
   this.resValve = 0;
+  //this.fieldValve = 0;
```

Comment:
This file has again some commented out extra lines. Please consider adding `@todo` to keep track of them.